app/uk/gov/hmrc/selfservicetimetopay/controllers/CalculatorController.scala

```diff
@@ -80,12 +80,16 @@ class CalculatorController(calculatorConnector: CalculatorConnector) extends Tim
   }
 
   def submitAmountsDue: Action[AnyContent] = Action.async { implicit request =>
+    var loggedIn = false
```

Comment:
No need for a var as far as I can see. Should evaluate directly on the presence of taxpayer within the ttpData as is done below @161